aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -18,6 +18,11 @@
 #ifdef USE_ROCM
 #define PYTORCH_ROCBLAS_VERSION_DECIMAL (ROCBLAS_VERSION_MAJOR * 100 + ROCBLAS_VERSION_MINOR)
 #define USE_GEMM_FLAGS_FP16_ALT_IMPL (PYTORCH_ROCBLAS_VERSION_DECIMAL >= 242)
+
+#if ROCM_VERSION >= 50200
+#include <rocsolver/rocsolver.h>
```

Comment:
Assuming this is intentional (instead of `hipsolver/hipsolver.h`), can you please point to a relevant JIRA that's related to the functionality that (I'm assuming) needs to be ported to hipsolver header?

Reply:
Just confirmed this is redundant and will be removed later.